src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { ResultsList, SearchBox } from './components';
+import './App.css';
+
+class App extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			results: [],
+		}
+	}
+
+	render() {
+		return (
+			<div className="container-fluid">
+				<div className="row">
+					<div className="col-xs-12">
+						<h1>Election Judge Lookup Tool</h1>
```

Comment:
Might be worth putting the app information in a smaller component.

Reply:
maybe AppHeader

Reply:
Is it work making a component if the thing is only used once, and doesn't have any logic?